diff --git a/api/app.ts b/api/app.ts
index 25465a5..1073160 100644
--- a/api/app.ts
+++ b/api/app.ts
@@ -78,7 +78,7 @@ apollo.applyMiddleware({ app })
 apollo.installSubscriptionHandlers(http)
 
 http.listen(PORT, () => {
-    console.log(`🚀 GraphQL service ready at http://localhost:${PORT}/graphql`)
+    console.log(`GraphQL service ready at http://localhost:${PORT}/graphql`)
 })
 
 runScheduler();
